pkg/chartutil/values.go

```diff
@@ -376,3 +379,39 @@ func istable(v interface{}) bool {
 	_, ok := v.(map[string]interface{})
 	return ok
 }
+
+// PathValue takes a yaml path with . notation and returns the value if exists
+func (v Values) PathValue(ypath string) (interface{}, error) {
+	if len(ypath) == 0 {
+		return nil, error(fmt.Errorf("yaml path string cannot be zero length"))
```

Comment:
You can remove the `error(...)` wrapper, since `fmt.Errorf` returns a type `error`.